### ISSUE TITLE
RFE: include sample dialog export build_vm_provision_request

### ISSUE THREAD

**Issue by jeffwarnica** [jeffmcutter/cf_shortcuts#21]
Trying the dialogs from (and older version of) cf_e, I hit some simple errors on a low effort deploy test.

A sample dialog export synced/matching the code would be helpful.

**Comment by jeffmcutter** [comment]
Makes sense.  Do you have one?  If not next time I use this I'll try to remember.
